Automattic/jetpack
# CRM: Changing location of externally added import Transaction menu links

## Proposed changes:

* This PR changes the location of the 'Import' link that is added when the CSV Pro extension is installed and active, under the 'Transactions' menu. Previously it was located at the bottom of the menu underneath a separator. Now it is located above the 'Export' link, to be more consistent with the placement in the Contact menu.
* If any extra menu items are added to the 'Transactions' menu via the `zbs-transactions-menu` filter, they will be added to the bottom of the menu underneath a separator.
* Another option was to make the menu / filter extensible in a way that would allow being able to add any new item at a specified location, but I've not taken that route (initially) for reasons mentioned here: https://github.com/Automattic/zero-bs-crm/issues/2896#issuecomment-1494617655

### Other information:


## Jetpack product discussion

https://github.com/Automattic/zero-bs-crm/issues/2896

## Does this pull request change what data or activity we track or use?

No.

## Testing instructions:

* To test this out, on a local development site without this branch checked out (or a Jurassic Ninja site with either the Jetpack CRM current version or even the Jetpack Beta Tester plugin active and set to bleeding edge or latest stable), make sure to add the CSV Pro extension and activate it. 
* The default behaviour when looking at the Transactions menu dropdown (from any CRM dashboard page) is that the 'Import' link will be at the bottom of the menu, underneath a separator.
* Checkout this branch locally or via the Beta Tester plugin and check the Transactions menu again - the 'Import' link should now be above the 'Export' link.

**Before:**
<img width="143" alt="Screenshot 2023-04-04 at 10 40 08" src="https://user-images.githubusercontent.com/16754605/229755020-2d4b08fd-c533-417d-a72b-9d0fb5a26ee5.png">


**After:**

<img width="155" alt="Screenshot 2023-04-04 at 10 38 27" src="https://user-images.githubusercontent.com/16754605/229755043-f50361b1-8f28-4730-8110-f801557a706c.png">
